stable/velero/templates/secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.credentials.useSecret (not .Values.credentials.existingSecret) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "velero.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "velero.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "velero.chart" . }}
+type: Opaque
+data:
+{{- range $key, $value := .Values.credentials.secretContents }}
```

Comment:
I wasn't able to get this format to work. I instead used the following, along with the associated command:

```
{{- if and .Values.credentials.useSecret (not .Values.credentials.existingSecret) -}}
apiVersion: v1
kind: Secret
metadata:
  name: {{ include "velero.fullname" . }}
  labels:
    app.kubernetes.io/name: {{ include "velero.name" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
    helm.sh/chart: {{ include "velero.chart" . }}
type: Opaque
data:
  cloud: |-
    {{ .Values.credentials.secretContents | b64enc }}
{{- end }}
```
`helm install --name velero --namespace velero -f values.yaml --set-file credentials.secretContents=/home/nrb/credentials-velero-gke ./velero`

Reply:
Grr, somehow github duplicated some review comments. Sorry!